sdk/playwrighttesting/microsoft-playwright-testing/src/utils/utils.ts

```diff
@@ -98,6 +99,20 @@ export const validateMptPAT = (
     if (result!.accountId !== claims!.aid) {
       validationFailureCallback(ServiceErrorMessageConstants.WORKSPACE_MISMATCH_ERROR);
     }
+    const currentTime = Date.now();
```

Comment:
Move expiration check logic in a separate function and keep it independent of validateMptPAT. validateMptPAT can be called in multiple places either today or in the future.